### ISSUE TITLE
show('midi') does not play any audio

### ISSUE THREAD

**Issue by edtky** [cuthbertLab/music21#332]
using a mac highSierra version 10.13.6 and running the following on Jupyter notebook:

an audio player emerges when I enter the following:
```
f = note.Note("F5")
f.show('midi')
```
However, there is no audio when I hit the play button.

I followed the recommendation to install QuickTime 7 in the documentation, but it still doesn't work:

> Maddeningly, Apple removed MIDI support in the version of QuickTime (QuickTime X) included in OS X 10.6 (Snow Leopard) and above (including Mountain Lion), so you’ll need to get the older QuickTime 7 from appleQuicktime to make that work.

**Comment by supersational** [comment]
Same here, seems it's a known issue with skipping the first note [here](https://groups.google.com/forum/#!topic/music21list/XsA4PRmqeOU).
It works if you add a rest as the first part of the score at least:
```python
import music21
f = music21.note.Note("F5")
score = music21.stream.Stream()
score.append(music21.note.Rest())
score.append(f)

score.show('midi')
```

**Comment by mscuthbert** [comment]
I *will* get to this patch eventually -- some MIDI players need the extra rest to not clip the first note and some people hate that it gets added when I used to automatically put it in.

**Comment by leoauri** [comment]
Hi Michael, everybody,
do you plan to patch for specific players, environments, and do you know which ones?
cheers, looks great,
Leo

**Comment by mscuthbert** [comment]
This has been fixed, I believe, in v6.3 -- please reopen if still a problem.
